backend/app/Http/Controllers/Api/CalendarApiController.php

```
@@ -0,0 +1,20 @@
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\CalendarActivity;
+
+class CalendarApiController extends Controller
+{
+
+    public function getCalendar()
+    {
+        return CalendarActivity::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\CalendarActivity' in method 'getCalendar'. (Clean Code Rules)

(StaticAccess)